config/initializers/active_admin.rb

```diff
@@ -153,7 +153,7 @@
   # To understand how to localize your app with I18n, read more at
   # https://github.com/svenfuchs/i18n/blob/master/lib%2Fi18n%2Fbackend%2Fbase.rb#L52
   #
-  config.localize_format = :long
+  # config.localize_format = :long
```

Comment:
Feature in `pre2`, might bring this back when we upgrade back even though we aren't using I18n.